README.md

```diff
@@ -18,6 +18,19 @@
 
 <hr />
 
+<div align="center">
```

Comment:
I find the alignment a bit weird now that there is more text, I'll wait for Elise review 👁️ 
<img width="864" alt="Screenshot 2021-07-08 at 09 51 36" src="https://user-images.githubusercontent.com/1931615/124883994-27df3d80-dfd2-11eb-81ff-27eab54d10ff.png">

Reply:
The centering causes a weird flow at certain widths, but the only alternative there is to remove the align attribute and let it align left 🤷🏻 ; otherwise for vertical spacing IMO it works better if you use a real heading tag and paragraphs, and remove the double line breaks, like this:

```md
<hr />

<div align="center">

#### 💡**Rasa Open Source 3.0 is coming up!**

[2.8](https://github.com/RasaHQ/rasa/milestone/39) will be the last minor in the 2.x series, as we need to pause releasing minors while we work on architectural changes in 3.0. You can still contribute new features and improvements which we plan to release together with 3.0. Read more about [our contributor guidelines](#how-to-contribute).

We plan to ship alpha releases and release candidates over the next few months in order to get early feedback. Stay tuned!

</div>

<hr />
```
![image](https://user-images.githubusercontent.com/1242864/125408773-897d1e80-e3bb-11eb-8fd9-fd675ae81d8b.png)